tests/system/small/test_session.py

```diff
@@ -390,9 +390,16 @@ def test_read_gbq_twice_with_same_timestamp(session, penguins_table_id):
     assert df3 is not None
 
 
-def test_read_gbq_on_linked_dataset_warns(session):
+@pytest.mark.parametrize(
+    "source_table",
+    [
+        "bigframes-dev.thelook_ecommerce.orders",
+        "bigframes-dev.bigframes_tests_sys.base_table_mat_view",
```

Comment:
I'm a concerned about hard coding this table. Could we at least include something in the `scripts/` directory that can recreate this table if needed?

Reply:
Got it, will add that.